PocketKit/Sources/Sync/RecentSavesWidgetStore.swift

```diff
@@ -0,0 +1,132 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// swiftlint:disable multiline_arguments_brackets
+import SharedPocketKit
+import WidgetKit
+
+enum RecentSavesStoreError: Error {
+    case invalidStorage
+}
+
+/// A store that provides a list of recent saves
+public protocol RecentSavesStore {
+    var isLoggedIn: Bool { get }
+    func setLoggedIn(_ isLoggedIn: Bool)
+    var recentSaves: [SavedItemContent] { get }
+    func updateRecentSaves(_ items: [SavedItemContent]) throws
+}
+
+/// A concrete implementation of `RecentSavesStore` used for the recent saves widget
+public struct RecentSavesWidgetStore: RecentSavesStore {
+    private static let isLoggedInKey = "RecentSavesWidgetLoggedInKey"
+    private static let recentSavesKey = "RecentSavesWidgetKey"
```

Comment:
There is a `UserDefaultsKey.swift` file in `SharedPocketKit`; it might be a good idea to move these keys there. That tends to be the master list of keys used throughout the project (and also adds a `resetKeys` function that's called on logout, which clears defaults "properly").

Reply:
👍